src/Raven.Server/Web/System/OngoingTasksHandler.cs

```diff
@@ -500,6 +478,60 @@ private void VerifyPeriodicBackupConfiguration(ref BlittableJsonReaderObject rea
             }
         }
 
+        [RavenAction("/databases/*/admin/backup-data-directory", "GET", AuthorizationStatus.DatabaseAdmin)]
+        public async Task FullBackupDataDirectory()
+        {
+            var path = GetStringQueryString("path", required: true);
+
+            string actualFullPath;
+
+            try
+            {
+                actualFullPath = GetActualFullPath(path, out _);
+            }
+            catch (Exception e)
+            {
+                actualFullPath = e.Message;
```

Comment:
What?!

Reply:
Example: Nodes A & B.
Path: E:\Temp (Exists on Node A but doesn't exist on Node B).

This will output the free space on Node A and output that the path doesn't exist on Node B.

Reply:
Not like this. Create a class that has the full path and an error, not overload the path in this manner.

Reply:
done